### PR TITLE
feat(NET): APM logs in context (ETA 04-04-22)

### PR DESCRIPTION
**Pull request by barbnewrelic**
Added new info

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx [hunk 1]
```diff
@@ -24,11 +24,125 @@ redirects:
...
+    title="Option 1 (simplest): Let the agent decorate and forward your logs."
+  >
+
+  This is the simplest approach, and it's a great choice for developers who may not have the access or interest in setting up a log forwarder, or for accounts that want to see the power of logs and other linking metadata in context of their apps, without a lot of overhead.
+  
+  Using this option, your logs will include `span.id`, `trace.id`, `hostname`, `entity.guid`, and `entity.name`. These attributes automatically link your logs to spans, traces, and other telemetry, making it easier to troubleshoot.
+
+  All you need to do is install an agent version with log forwarding capabilities ([.NET agent 9.7.0 or higher](/docs/release-notes/agent-release-notes/net-release-notes)), and update your configuration to enable forwarding.
+
+  **INSERT .NET-specific configuration**
```
Comment (jaffinito) [inline src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx:60]: This block is the simplest form of enabling log forwarding using the newrelic.config file.

```xml
<applicationLogging enabled="true">
 <logForwarding enabled="true/>
</applicationLogging>
```

File: src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx [hunk 2]
```diff
@@ -24,11 +24,125 @@ redirects:
...
+
+  This option may be on by default in a future agent version.
+  
+  **Optional adjustments:**
+
+  Once this is enabled, you also have control over the maximum number of logs sent to New Relic every minute. The default value is 10,000. If more than 10,000 logs are received in a 60-second window, your logs will begin to be sampled.
+  
+  Set it to a higher number to receive more logs. Set it to a lower number to receive fewer logs. Any integer is valid.
+
+  **INSERT .NET-specific configuration**
```
Comment (jaffinito) [inline src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx:72]: This block adds in the maxSamplesStored setting that customers can alter.

```xml
<applicationLogging enabled="true">
 <logForwarding enabled="true" maxSamplesStored="10000" />
</applicationLogging>
```

File: src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx [hunk 3]
```diff
@@ -24,11 +24,125 @@ redirects:
...
+
+  Already have a log forwarder you like? We've got you covered! Language agents can decorate your logs with the linking metadata needed to provide access to automatic logs-in-context features.
+
+  1. If you are currently sending logs using the old logs-in-context method, be sure to remove that code from your [specific log forwarder](/docs/logs/forward-logs/enable-log-management-new-relic#log-forwarding) before enabling local decoration. Otherwise, your app may send double log lines. Depending on your account, this could result in double billing.
+
+  2. Enable log decorating in your configuration, then relaunch the agent to start decorating your logs.
+  
+  
+
+  **INSERT .NET-specific configuration**
```
Comment (jaffinito) [inline src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx:91]: This block shows how to enable local decorating, but actually getting the logs decorated requires manual changes to the customers config framework.  Do we want to link the .NET specific docs for the specifics or put all that in here?

```xml
<applicationLogging enabled="true">
 <localDecorating enabled="true" />
</applicationLogging>
```

Comment (barbnewrelic) [reply src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx:91]: I've reworded this to follow how Ruby handled this. OK by you?

**Comment by barbnewrelic** [comment]
.NET is ready for release

**Pull request merged**